src/main_app.jl

```diff
@@ -101,34 +97,33 @@ function _indicative_result_display(res)
     upper = Y_no[:upper_95]
     lower = Y_no[:lower_95]
 
-    p = plot(upper, fillrange=lower, color=:lightsalmon1, alpha=0.8, label="")
-    plot!(Y_no[:median], label="No Deployment median", linecolor=:red, alpha=0.8)
-
+    p = plot(upper; fillrange=lower, color=:lightsalmon1, alpha=0.8, label="")
+    plot!(Y_no[:median]; label="No Deployment median", linecolor=:red, alpha=0.8)
 
     # Unguided Deployment
     upper = Y_ung[:upper_95]
     lower = Y_ung[:lower_95]
-    p = plot!(upper, fillrange=lower, color=:lightblue2, alpha=0.8, label="")
-    plot!(Y_ung[:median], label="Unguided median", linecolor=:blue, alpha=0.5)
-
+    p = plot!(upper; fillrange=lower, color=:lightblue2, alpha=0.8, label="")
+    plot!(Y_ung[:median]; label="Unguided median", linecolor=:blue, alpha=0.5)
 
     # Guided
     upper = Y_g[:upper_95]
     lower = Y_g[:lower_95]
 
-    plot!(upper, fillrange=lower, color=:lightseagreen, alpha=0.4, label="")
-    plot!(Y_g[:median],
+    plot!(upper; fillrange=lower, color=:lightseagreen, alpha=0.4, label="")
+    plot!(Y_g[:median];
         label="Guided median", linecolor=:green, alpha=0.4,
         xlabel="Year", ylabel="Relative Cover",
         xticks=(1:75, year_axis))
 
-    p2 = plot(Y_ung[:mean] - Y_no[:mean], label="Guided - No Deployment (μ)",
+    p2 = plot(Y_ung[:mean] - Y_no[:mean]; label="Guided - No Deployment (μ)",
         xlabel="Year", ylabel="δ Relative Cover",
         xticks=(1:75, year_axis), color=:red
     )
```

Comment:
```suggestion
    p2 = plot(
        Y_ung[:mean] - Y_no[:mean]; 
        label="Guided - No Deployment (μ)",
        xlabel="Year", 
        ylabel="δ Relative Cover",
        xticks=(1:75, year_axis), 
        color=:red
    )
```

Reply:
This `main_app` file is marked for removal. It was an earlier attempt to spin up a desktop GUI but isn't relevant for us any more.